backend/src/modules/authentication/core/application/command/SetPasswordCommandHandler.ts

```diff
@@ -5,6 +5,7 @@ import { CommandResult } from '../../../../../shared/core/application/command/Co
 import { SetPassword } from './SetPassword';
 import { AuthenticationRepository } from '../AuthenticationRepository';
 import { setPasswordForUserAccount } from '../../domain/UserAccount';
+import bcrypt from 'bcrypt';
 
 export class SetPasswordCommandHandler implements CommandHandler<SetPassword> {
```

Comment:
Aaa, w ogole kiedy to bedzie uzywane, mi przypomnij :) ? Jaki jest use case tego?

Reply:
eee jeszcze nie, ale pewnie będzie ^_^, chyba fajnie mieć możliwość zmiany hasło :-P

Reply:
było w miro i to zaczęła Ania wgl robić, a ja tam trochę grzebię